package.json

```diff
@@ -14,28 +14,32 @@
     "2d",
     "mobile"
   ],
+  "author": "Ali Shakiba",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/shakiba/planck.js.git"
   },
-  "main": "dist/planck.js",
-  "types": "index.d.ts",
-  "author": "Ali Shakiba",
   "contributors": [
     "Erin Catto",
     "Ali Shakiba",
     "Oliver Zell"
   ],
-  "dependencies": {
-    "stage-js": "^0.9.4"
+  "type": "module",
+  "types": "./index.d.ts",
+  "main": "./dist/planck.umd.cjs",
```

Comment:
csj?

Reply:
I think for nodes we can remove csj (dead since node 12 died) I'm also not sure if main has to be esm if type is module